Composer/plugins/azurePublish/yarn.lock

```diff
@@ -178,11 +179,19 @@
 "@bfc/shared@../../packages/lib/shared":
   version "0.0.0"
   dependencies:
+    "@botframework-composer/types" "*"
     format-message "6.2.3"
     json-schema "^0.2.5"
     nanoid "^3.1.3"
     nanoid-dictionary "^3.0.0"
 
+"@botframework-composer/types@*", "@botframework-composer/types@0.0.1":
+  version "0.0.1"
+  resolved "https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/@botframework-composer/types/-/@botframework-composer/types-0.0.1.tgz#117b86bf8e3bf43567471d436eb1c2f0ffcd6bb8"
```

Comment:
is this expected?

Reply:
Which part? The registry?

Reply:
yeah the registry

Reply:
There is an `.npmrc` in this directory that sets that registry. so yes?

Reply:
didn't see that